lua/entities/ent_jack_aidfuel_propane/init.lua

```diff
@@ -45,23 +31,4 @@ end
 
 function ENT:OnTakeDamage( dmginfo )
     self:TakePhysicsDamage( dmginfo )
-    self.StructuralIntegrity = self.StructuralIntegrity - dmginfo:GetDamage()

```

Comment:
This isn't used anywhere else?

Reply:
Nope, good catch though i found other stuff that defines it for no reason